inject-groovy/src/test/groovy/io/micronaut/inject/field/FieldInjectionSpec.groovy

```diff
@@ -66,8 +63,8 @@ class FieldInjectionSpec extends Specification {
 
         then:
         e.d == null
-        e.value == "Default greeting"
-        e.property == "Default greeting"
+        e.value == null
```

Comment:
I think we should maintain this behavior if possible

Reply:
That’s only Groovy related, Java code is returning nulls.
Do we want to have those inconsistencies?

Reply:
No all good then. Should make a note in the breaks.adoc about it though

Reply:
I have added the note to the breaking changes